spi/eink_mono_spi/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(eink_mono_spi
+        eink_mono_spi.c
+        )
+
+# Pull in our (to be renamed) simple get you started dependencies
```

Comment:
Uh-oh :laughing:

Reply:
OMG. I think something similar happened and that's when I decided to submit the other two PRs.